charts/newrelic-infrastructure/README.md

```diff
@@ -50,7 +50,7 @@ This chart will deploy the New Relic Infrastructure agent as a Daemonset.
 | `schedulerEndpointUrl`         | Explicitly sets the scheduler component url.                                                                                                                                                                                                      |                                 |
 | `controllerManagerEndpointUrl` | Explicitly sets the controller manager component url.                                                                                                                                                                                             |                                 |
 | `eventQueueDepth`              | Increases the in-memory cache of the agent to accommodate for more samples at a time. | |
-| `enableProcessMetrics`         | Enables the sending of process metrics to New Relic.  | `false` |
+| `enableProcessMetrics`         | Enables the sending of process metrics to New Relic.  | `` |
```

Comment:
I'd maybe puy a "See [1]" in the "Default" column, with a short explanation after the table (basically the same or a shorter version of what you put in the values.yml)

Reply:
Something like:

 | `enableProcessMetrics`         | Enables the sending of process metrics to New Relic.  | Account default[1] | 

> [1]: The account default is `true` for accounts created before July 20, 2020 and `false` for accounts created after said date. Setting this to `true` or `false` will override this mechanism.

Reply:
I'll add a note!